src/mcp-server.ts

```
@@ -111,173 +140,248 @@ export class AzureDevOpsMCPServer {
-      ]
-    };
+          text: JSON.stringify(items, null, 2),
+        },
+      ],
+    }
   }
 
-  private async handleSyncData() {
-    await this.syncService.performSync();
-    
+  private async handleSyncData(args: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)